layouts/shortcodes/table.html

```diff
@@ -1,6 +1,11 @@
 {{ $hasCaption := isset .Params "caption" }}
 {{ $caption    := .Get "caption" }}
+{{ $sortable   := .Get "sortable" }} 
 {{ $captionEl  := printf "<table><caption style=\"display: none;\">%s</caption>" $caption }}
 {{ $table      := .Inner | markdownify }}
 {{ $html       := cond $hasCaption ($table | replaceRE "<table>" $captionEl) $table | safeHTML }}
+<!-- Check if 'sortable' is true, and if so, add the 'sortable-table' class -->
+{{ if $sortable }}
+    {{ $html = replaceRE "<table>" "<table class=\"sortable-table\">" $html | safeHTML }}
+{{ end }}
```

Comment:
Try this (untested):
```suggestion
{{ $tableElementExtraString = cond $sortable " class=\"sortable-table\"" "" }}
{{ $captionElementString = cond $hasCaption (printf "<table><caption style=\"display: none;\">%s</caption>" $caption | markdownify ) "" }}
{{ $html = replaceRE "<table>" ( printf "<table%s>%s" $tableElementExtraString $captionElementString ) $html | safeHTML }}
```

I haven't checked this revised code, so please only accept the suggestion if you know it does actually work.

Reply:
That's to fix a new problem, which is that tables that are sortable _and_ have captions now won't work.

Reply:
In the [preview page](https://deploy-preview-42412--kubernetes-io-main-staging.netlify.app/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features), the feature gate table functions as expected with its sortable feature and includes a caption. 

Taking a look this feedback request, it appears that there is an issue with the suggested code, but will investigate and address it.

Reply:
Nope, I'm wrong. This algorithm (that you've proposed) is correct with no changes.